src/app/core/containers/app.component.ts

```diff
@@ -6,11 +6,13 @@ import { Platform } from 'ionic-angular'
 
 import { SplashPageComponent } from '../../pages/splash/containers/splash-page.component'
 import { NotificationService } from '../services/notifications/notification.service'
+import {ProgressReportPageComponent} from "../../pages/progress-report/containers/progress-report-page.component";
 
 @Component({
   template: '<ion-nav *ngIf="isAppInitialized" [root]="rootPage"></ion-nav>'
 })
 export class AppComponent {
+  // rootPage = ProgressReportPageComponent
```

Comment:
Should this be removed or is this used for testing?

Reply:
Yes. I should remove this comment and also revert back the defaultConfig to the original one. Now It's changed to our test platform and my protocol repo.